pep_sphinx_extensions/pep_processor/parsing/pep_banner_directive.py

```diff
@@ -109,18 +109,20 @@ class CanonicalTypingSpecBanner(PEPBanner):
 
     admonition_pre_template = (
         "This PEP is a historical document. "
-        "The up-to-date, canonical spec, {link_content}, is maintained on "
-        f"the `typing specs site <{TYPING_SPEC_BASE_URL}>`__."
+        "The up-to-date, canonical specs, {link_content}, are maintained at "
+        f"the `typing specs site <{TYPING_SPEC_BASE_URL}>`__ and the CPython "
```

Comment:
the "and" here doesn't quite read correctly to me. E.g. it could read as implying the CPython documentation contains the information in the typing specs (also I don't know how much of the CPython docs I'd treat as specification)

Maybe a separate sentence would be better: "The CPython documentation describes runtime behaviour"

Reply:
> I don't know how much of the CPython docs I'd treat as specification

you could argue they "specify" (or _should_ specify?) the behaviour for the runtime objects -- but yeah, agree it's probably a stretch! Will reword.

Reply:
Pushed a rewording in https://github.com/python/peps/pull/3821/commits/6410ef123295e81b92aa03151ca199775c7b295e -- how's it look now? Preview examples:
- https://pep-previews--3821.org.readthedocs.build/pep-0526/
- https://pep-previews--3821.org.readthedocs.build/pep-0561/